zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/behavior/BpmnBehaviors.java

```diff
@@ -55,4 +56,6 @@ public interface BpmnBehaviors {
   BpmnUserTaskBehavior userTaskBehavior();
 
   BpmnCompensationSubscriptionBehaviour compensationSubscriptionBehaviour();
+
+  JobUpdateBehaviour jobUpdateBehaviour();
```

Comment:
I thought so as well but it actually doesn't really look like it, to be honest. But it might also be something we need to clean up, so those behaviors shouldn't be called `BpmnJob...Behavior`. I am not entirely sure to be honest.